src/NServiceBus.AcceptanceTests/EndpointTemplates/DefaultServerWithDiagnostics.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace NServiceBus.AcceptanceTests.EndpointTemplates
+{
+    class DefaultServerWithDiagnostics : DefaultServer
+    {
+        public DefaultServerWithDiagnostics()
+        {
+            DisableDiagnostics = false;
```

Comment:
Seems that previously we used aggregation instead of inheritance https://github.com/Particular/NServiceBus/blob/develop/src/NServiceBus.AcceptanceTests/EndpointTemplates/DefaultPublisher.cs#L11